Documentation/k8s-pre-reqs.md

```diff
@@ -9,19 +36,21 @@ export KUBECONFIG="$(pwd)/kubeconfig"
 kubectl config use-context vagrant-multi
 ```
 
-Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
+Then wait for the cluster to come up and verify that Kubernetes is done initializing (be patient, it takes some time):
 
 ```
 kubectl cluster-info
 ```
 
-Once you see a url response, your cluster is [ready for use by Rook](kubernetes.md#deploy-rook).
+Once you see an URL response, your cluster is [ready for use by Rook](kubernetes.md#deploy-rook).
```

Comment:
grammar nit: It should be "a URL" (based on the pronunciation of the individual letters "you are ell", which doesn't start with a vowel sound, so it is "a", not "an")